File: src/main/java/com/market/ddanggyeo/redis/hashEntity/TokenHash.java
package com.market.ddanggyeo.redis.hashEntity;

import lombok.Getter;
import lombok.Setter;
import org.springframework.data.redis.core.RedisHash;

import javax.persistence.Id;
import java.io.Serializable;

@Getter
@Setter
@RedisHash("refreshToken")
public class TokenHash implements Serializable {
    @Id
    private String id;
    private String email;
    private String refreshToken;
}

File: src/main/kotlin/com/market/ddanggyeo/redis/hashEntity/TokenHash.kt
package com.market.ddanggyeo.redis.hashEntity

import org.springframework.data.redis.core.RedisHash
import javax.persistence.Id

@RedisHash("refreshToken")
data class TokenHash(
    @Id
    val id: String? = null,
    val email: String? = null,
    val refreshToken: String? = null
)
